core/src/main/kotlin/com/justai/jaicf/slotfilling/SlotFillingBotContextExt.kt

```diff
@@ -1,15 +1,57 @@
 package com.justai.jaicf.slotfilling
 
+import com.justai.jaicf.BotEngine
+import com.justai.jaicf.activator.ActivationContext
+import com.justai.jaicf.activator.Activator
+import com.justai.jaicf.context.ActivatorContext
 import com.justai.jaicf.context.BotContext
+import com.justai.jaicf.model.activation.Activation
 
-internal const val SLOTFILLING_ACTIVATOR_KEY = "com/justai/jaicf/slotfilling/activator"
+private const val SLOTFILLING_ACTIVATOR_KEY = "com/justai/jaicf/slotfilling/activator"
+private const val SLOTFILLING_NEXT_STATE_KEY = "com/justai/jaicf/slotfilling/nextState"
+
+internal data class SlotFillingContext(
+    val activator: Activator,
+    val nextState: String,
```

Comment:
What is the purpose of this prop? Looks like it's unused anywhere.

Reply:
It's used inside `finishSlotFilling`.
In the previous version, we didn't store `nextState` in context, so there was a hack when we set `dialogContext.nextState` on the first activation and finally create `Activation` with saved `nextState = dialogContext.nextState`. Now it looks more clear.